### ISSUE TITLE
Several anchors reference non-existent sections

### ISSUE THREAD

**Issue by EraYaN** [jellyfin/jellyfin-docs#116]
The DocFX compiler reports the following:

```
[19-09-11 09:23:12.019]Warning:[BuildCommand.Postprocess.HandlePostProcessorsWithIncremental.HandlePostProcessors.Processing html.ValidateBookmark](docs/faq.md#L28)Invalid link: '[requesting features](xref:contrib-issues#requesting-features)'. The file docs/contributing/issues.md doesn't contain a bookmark named 'requesting-features'.
[19-09-11 09:23:12.020]Warning:[BuildCommand.Postprocess.HandlePostProcessorsWithIncremental.HandlePostProcessors.Processing html.ValidateBookmark](index.md#L36)Invalid link: '[Docker](xref:admin-installing#docker-hub)'. The file docs/administration/installing.md doesn't contain a bookmark named 'docker-hub'.
[19-09-11 09:23:12.020]Warning:[BuildCommand.Postprocess.HandlePostProcessorsWithIncremental.HandlePostProcessors.Processing html.ValidateBookmark](index.md#L63)Invalid link: '[Requesting Features](xref:contrib-issues#requesting-features)'. The file docs/contributing/issues.md doesn't contain a bookmark named 'requesting-features'.
```

**Comment by theg00s3** [comment]
As of today, these still exist:
`[19-12-13 11:13:20.802]Warning:[BuildCommand.BuildCore.Build Document.LinkPhaseHandlerWithIncremental.Apply Templates](general/administration/connectivity.md)1 invalid cross reference(s) "[port forwarding](xref:admin-port-forwarding)".
[19-12-13 11:13:21.085]Warning:[BuildCommand.Postprocess.HandlePostProcessorsWithIncremental.HandlePostProcessors.Processing html.ValidateBookmark](general/faq.md#L28)Invalid link: '[requesting features](xref:contrib-issues#requesting-features)'. The file general/contributing/issues.md doesn't contain a bookmark named 'requesting-features'.
[19-12-13 11:13:21.087]Warning:[BuildCommand.Postprocess.HandlePostProcessorsWithIncremental.HandlePostProcessors.Processing html.ValidateBookmark](index.md#L36)Invalid link: '[Docker](xref:admin-installing#docker-hub)'. The file general/administration/installing.md doesn't contain a bookmark named 'docker-hub'.
[19-12-13 11:13:21.089]Warning:[BuildCommand.Postprocess.HandlePostProcessorsWithIncremental.HandlePostProcessors.Processing html.ValidateBookmark](index.md#L39)Invalid link: '[Windows](xref:admin-installing#windows-x64x86)'. The file general/administration/installing.md doesn't contain a bookmark named 'windows-x64x86'.
[19-12-13 11:13:21.090]Warning:[BuildCommand.Postprocess.HandlePostProcessorsWithIncremental.HandlePostProcessors.Processing html.ValidateBookmark](index.md#L63)Invalid link: '[Requesting Features](xref:contrib-issues#requesting-features)'. The file general/contributing/issues.md doesn't contain a bookmark named 'requesting-features'.`

**Comment by theg00s3** [comment]
These are fixed in #173
